HashRegistrarSimplified.sol

```diff
@@ -0,0 +1,340 @@
+
+
+
+/*
+
+Temporary Hash Registrar 
+========================
+
+This is a simplified version of a hash registrar. It is purporsefully limited:
+names cannot be six letters or shorter, new auctions will stop after 4 years
+and all ether still locked after 8 years will become unreachable.
+
+The plan is to test the basic features and then move to a new contract in at most
+2 years, when some sort of renewal mechanism will be enabled.
+
+*/
+
+import 'ENS.sol';
+
+
+contract Deed {
+    /* 
+    The Deed is a contract intended simply to hold ether
+    It can be controlled only by the registrar and can only send ether back to the owner.
+    */
+    Registrar public registrar;
```

Comment:
Actually, this can be type address; it never calls any methods on the registrar.